tools/scripts/verification.scripts/ResourceStringCommentVerification.ps1

```diff
@@ -0,0 +1,68 @@
+﻿# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+<#
+.SYNOPSIS
+Verifies that all .resx files in the target folder have comments for resource strings.
+
+.PARAMETER Target
+The target folder to search recursively.
+
+.Example Usage 
+.\ResourceStringCommentVerification.ps1 -Target 'C:\code\axe-windows\src'
+#>
+
+param(
+    $Target
+)
+
+$NewLine=([Environment]::NewLine)
+$FailedStrings = @()
+# Exclude the projects that are not yet 
+$excludeList="(Actions|Automation|CLI|Desktop|tools)"
```

Comment:
Might it be better to have a whitelist, explicitly **including** namespaces that we're localizing? Can we keep this list in one place that everything else can reference?

Reply:
My thought with having an exclude list is that, if/when we add new namespaces in the future, they will automatically be included in this check. This may or may not be the behavior we want for new namespaces, but it would force us to make an explicit decision, as opposed to just forgetting about translator comments for a new namespace altogether. 

> Can we keep this list in one place that everything else can reference?

Where else would we be referencing this?

Reply:
> My thought with having an exclude list is that, if/when we add new namespaces in the future, they will automatically be included in this check.

Makes sense to me.

> Where else would we be referencing this?

In whatever automatically builds the localization catalogues.